packages/session/index.js

```diff
@@ -115,7 +115,8 @@ const session = (req, res) => {
 
 const createSessionMiddleware = (
   { signinPath, signoutPath, sessionPath, successPath },
-  authStrategy
+  authStrategy,
+  audiences
```

Comment:
What's the point of passing this in here? It'd be hard-coded on app start-up, which means it'd be the same for everyone.

Perhaps this is placeholder until we bottom out on an API for specifying what users are in what audiences?

If so, I'd say we don't expose it at this level, instead, we leave it off, and just pass `[]` below in the `signin()` call.

Reply:
`createSessionMiddleware` can potentially be called multiple times by multiple independent concerns. We currently only call it once, but that will change in the very near future.